plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerBehaviour.java

```diff
@@ -59,6 +61,16 @@ public void stop(boolean force) {
     return super.getResources(module);
   }
 
+  @Override
+  public IStatus canStop() {
```

Comment:
I need to be able to access the server to change the server state and that is proving to be very painful.